deploy/helm/lunettes/templates/grafana/grafana-cm.yaml

```diff
@@ -2609,7 +2609,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "Pod信息",
-                        "url": "\r\nhttp://lunettes.lunettes.svc.cluster.local:8080/api/v1/debugpod?uid=${__data.fields.PodUid}"
```

Comment:
there's also a few other places using `lunettes.lunettes.svc.cluster.local:8080`, can you please help to update those links as well @LLKCoder

Reply:
okay